cluster/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
how is this different than the current "docker-build" target we have? isn't there an overlap?

Reply:
I wanted to keep it in Makefile, but I did not figure out how to `source` and obtain the port. Any idea how to fix it?